nifti2/clib_02_nifti2.c

```diff
@@ -19,7 +19,7 @@
 #include <nifti2_io.h>
 int disp_float(nifti_image * nim, int vol, int slice, int line, int offset);
 
-int show_help( void )
+static int show_help( void )
```

Comment:
this is a main program, where static should not really matter

Reply:
Indeed, there should be no change in behaviour here.

The motivation is to fix all `-Wmissing-prototypes` warnings because that warning in general can and does find real bugs.

It's just good practice for symbols not needed outside the translation unit to be declared `static`.